src/pages/Hwangsoohyeon/Join/Join.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import './Join.scss';
+import { useNavigate } from 'react-router-dom';
```

Comment:
import 순서 컨벤션 가이드 다시 알려드리겠습니다!!
1. 모듈
2. 컴포넌트
3. css파일 
순서 생각해서 수정해주세요! 
```suggestion
import React, { useState } from 'react';
import { useNavigate } from 'react-router-dom';
import './Join.scss';
```